src/components/MarkdownProvider/components/CodeExample/utils/retrieveCodesandboxParameters.ts

```diff
@@ -56,6 +56,7 @@ const packageJson = `
     "react": "latest",
     "react-dom": "latest",
     "styled-components": "latest",
+    "lodash.debounce": "latest",
```

Comment:
The `Multiselect` examples use debounce, but the other examples on the website do not. In the future, we could update the `CodeExample` component to take in specific packages. But I think this works for now.